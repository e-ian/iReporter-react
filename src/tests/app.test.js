import React from "react";
import renderer from "react-test-renderer";
import App from "../containers/app";
// import LoginComponent from "../components/LoginComponent";

describe("App", () => {
  it("should render without crashing", () => {
    const wrapper = renderer.create(<App />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
  // it("should render the login component", () => {
  //   const wrapper = renderer.create(<LoginComponent />);
  //   expect(wrapper.toJSON()).toMatchSnapshot();
  // });
});
