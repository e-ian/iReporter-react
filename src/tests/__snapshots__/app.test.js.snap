// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without crashing 1`] = `
Array [
  <div
    className="Toastify"
  />,
  <div>
    <div
      className="wrapper"
    >
      <div
        className="welcome"
      >
        <div
          className="report"
        >
          <h1>
            <a
              className="logo"
              href="/"
            >
              iReporter!
            </a>
          </h1>
        </div>
        <p>
          <em>
            Breaking the corruption chain together
          </em>
        </p>
        <p>
          <em>
            login to report an incident
          </em>
        </p>
        <p>
          <a
            className="cat-btn"
            href="login"
          >
            login
          </a>
        </p>
        <h3>
          <a
            href="signup"
          >
            SignUp here
          </a>
        </h3>
      </div>
    </div>
  </div>,
]
`;
